### ISSUE TITLE
Shutting down a ketrew server fails when the configuration file is missing

### ISSUE THREAD

**Issue by ihodes** [hammerlab/ketrew#262]
```
[hodesi01@hammerlab-dev2 ~]$ ketrew stop -P daemon
ketrew: internal error, uncaught exception:
        Sys_error("/hpc/users/hodesi01/.ketrew/configuration.json: No such file or directory")
```

**Comment by smondet** [comment]
Well that makes sense :)
We need the configuration of that particular server to know how to shut it down.

If you have removed the configuration, you recreate it or do a mass killing of `ketrew` processes.

**Comment by ihodes** [comment]
Fair point. Would it be irresponsible for ketrew to offer to kill ketrew processes under the current user?

**Comment by smondet** [comment]
You mean reimplementing `killall ketrew`?

I use this https://gist.github.com/smondet/7345ba91783688679fb0 instead of `killall`  because I want to be asked for confirmation.
